sfepy/terms/terms_surface.py

```diff
@@ -113,7 +113,7 @@ class SDLinearTractionTerm(Term):
     :Definition:
 
     .. math::
-        \int_{\Gamma} \ul{v} \cdot \ull{\sigma} \cdot \ul{n},
+        \int_{\Gamma} \ul{v} \cdot (\ull{\sigma}\, \ul{n}),
```

Comment:
`\cdot` is at many places, is this needed? It is not limited to dot of vectors in the docs...

Reply:
I would use `\cdot` only for the scalar product. Otherwise, the equations will be hard to read.

Reply:
IIRC it was used as "contraction" (matrix multiplication) as opposed to tensor product (`a \cdot b` having lower dimension as opposed to `a b` having higher). IMO `(,)` is also a notation for the dot product of "vectors" which is not the case here. The clearest might be just to use the Einstein summation convention and indices :) Anyway, revisiting the overall term notation is for another PR.
(Alternative: `\ul{v} \cdot \ull{\sigma} \ul{n}`)